app/presenters/result_set_presenter.rb

```diff
@@ -10,14 +10,14 @@ class ResultSetPresenter
            :atom_url,
            to: :finder
 
-  def initialize(finder, filter_params, view_context, sort_presenter, show_top_result = false)
+  def initialize(finder, filter_params, sort_presenter, next_and_prev_links, show_top_result = false)
```

Comment:
it's great that the view_context has been removed here! 
On the other hand there seem to be more and more parameters added to this constructor. Is there something we can do about this? - or should do  another refactor to clean that up?